### PR TITLE
Azure Service Bus "target"and "source" integration example and guidance

### PR DESCRIPTION
**Pull request by kaizimmerm**
Hi,

a similar sample then #11 but with Azure Service Bus which due to its queue support without partitioning will allow Ditto to consume messages as well.

I introduced a parent POM as well as this is now the second sample.

As always, let me know what you think or even better give it a try 😄

Kai

Signed-off-by: Kai Zimmermann <kai.zimmermann@microsoft.com>

### CODE REVIEW

File: azure/azure-servicebus-amqp10/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019 Microsoft.
```
Comment (thjaeckle) [inline azure/azure-servicebus-amqp10/pom.xml]: Hi Kai.
Could you please use the header template: https://github.com/eclipse/ditto-examples/blob/master/legal/license-header.txt

The copyright holders are listed in the NOTICE file: https://github.com/eclipse/ditto-examples/blob/master/legal/NOTICE.md

File: azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/DittoAzureServiceBusExampleApplication.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Microsoft
```
Comment (thjaeckle) [inline azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/DittoAzureServiceBusExampleApplication.java]: please adjust header to template

File: azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/MessageConsumer.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 Microsoft
```
Comment (thjaeckle) [inline azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/MessageConsumer.java]: please adjust header to template

File: azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/MessageProducer.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019 Microsoft
```
Comment (thjaeckle) [inline azure/azure-servicebus-amqp10/src/main/java/org/eclipse/ditto/examples/azure/MessageProducer.java]: please adjust header to template

File: azure/pom.xml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019 Microsoft.
```
Comment (thjaeckle) [inline azure/pom.xml]: please adjust header to template

File: azure/azure-servicebus-amqp10/pom.xml (hunk 1 above)
Comment (kaizimmerm) [reply azure/azure-servicebus-amqp10/pom.xml]: done

**Comment by thjaeckle** [comment]
Thx for your PR :+1:

**Pull request merged**